docs-src/dev-guide/major-components.md

```diff
@@ -0,0 +1,263 @@
+---
+id: major-components
+title: Temporal SDKs major components
+description: Temporal SDKs provide developers with a Temporal Client to communicate with a Temporal Cluster, APIs to define Workflows, APIs to configure and start Workers, APIs to customize Activity Executions, and a common library for code that’s used across your application.
+sidebar_label: Major components
+tags:
+  - temporal sdk
+  - explanation
+  - components
+---
+
+**What are the major components of Temporal SDKs?**
+
+Temporal SDKs offer developers with the following:
+
+- A Temporal Client to communicate with a Temporal Cluster
+- APIs to develop application code Workflows & Activities
+- APIs to configure and run Workers
+- A common library for code that’s used across the Client, Worker, and Workflow
```

Comment:
This is slightly confusing because it reads like it's describing something distinct from the other bulletpoints, but I don't think it actually is.